tests/destinations/test_custom_destination.py

```diff
@@ -58,10 +61,55 @@ def items_resource() -> TDataItems:
 
     p = dlt.pipeline("sink_test", destination=test_sink, dev_mode=True)
     p.run([items_resource()])
-
     return calls
 
 
+def test_custom_destination_info():
+    @dlt.destination
+    def test_sink(items: TDataItems, table: TTableSchema) -> None:
+        pass
+
+    p = dlt.pipeline("sink_test", destination=test_sink, dev_mode=True)  # type: ignore
+    assert p.destination.destination_info == {
+        "destination_name": "test_sink",
+        "destination_type": "dlt.destinations.destination",
+        "environment": None,
+        "fingerprint": digest128("test_sink"),
+        "repr": "test_sink(dlt.destinations.destination)",
+    }
+
+
+@pytest.mark.parametrize("destination_name", IMPLEMENTED_DESTINATIONS)
+def test_destination_info(destination_name):
```

Comment:
this is a good test. it should go to `tests/common/test_destinations.py`. I'm not sure it will pass - you create configs and sometimes this requires destination specific deps.

pls. fix the PR and let's see if tests are passing

Reply:
Done